puzzlejs/puzzle.css

```diff
@@ -26,9 +26,18 @@
     user-select: none;
 }
 
+.puzzle-entry {
+    --cell-size: 40px;
+}
+
+.puzzle-entry tr {
+    display: block;
+    height: var(--cell-size);
+}
+
 .puzzle-entry td {
-    width: 40px;
-    height: 40px;
+    width: var(--cell-size);
+    height: var(--cell-size);
     padding: 0px;
     position: relative;
     background: transparent;
```

Comment:
I wasn't able to leave a comment on the correct line since it's not "part of the diff". But you might try something like

```
font-size: calc(min(var(--cell-size)*.8, 32px))
```

Then someone who changes `--cell-size` to make it smaller but neglects to change `font-size` will get a smaller font size automatically! 

You can drop the `min` part if you want the font to scale up alongside larger cell sizes—I wasn't sure what you thought would be the right choice there.

Reply:
Great idea, thanks!